src/openklant2/tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from openklant2.tests.helpers import OpenKlantService
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--with-openklant-service",
+        action="store_true",
+        default=False,
+        help="Whether to seed a fresh OpenKlant Docker instance for each test",
+    )
+
+
+@pytest.fixture(scope="session", autouse=True)
+def openklant_service_singleton(request):
+    service = OpenKlantService()
+    if request.config.getoption("--with-openklant-service"):
+        with service as running_service:
+            yield running_service
+    else:
+        yield service
+
+
+@pytest.fixture()
+def client(request, openklant_service_singleton: OpenKlantService):
+    if request.config.getoption("--with-openklant-service"):
```

Comment:
Since we already perform this check in `open_klant_service_singleton`, is there a way to avoid doing it again here?